lib/src/clojure_lsp/queries.clj

```diff
@@ -249,7 +249,9 @@
       ;; maybe loaded by :require-macros, in which case, def will be in a clj file.
       (let [definition (find-definition db (assoc var-usage :lang :clj))]
         (when (:macro definition)
-          definition)))))
+          definition)))
+    ;; fallback to navigate from clojure to clojurescript vars
+    (find-definition db (assoc var-usage :lang :cljs))))
```

Comment:
I can't easily understand why this is important, AFAIK a clj code is not possible to call cljs ones right? only cljc -> cljs.
Also, the when above already handles cljs

Reply:
Maybe create a test that makes a little bit clear this?

Reply:
@mk will explain the details, but try this branch of clerk:

https://github.com/nextjournal/clerk/tree/jump-to-definition

Go to the `notesbooks/jump_definition.clj` notebook and try navigation from inside of the quoted function. You will see that it doesn't work. In this PR we added a fallback for this with the idea of navigating to something is better than nothing.

I used `bb debug-cli` btw and it was very useful, thank you.

Reply:
Cool :)
I understood the idea, I just have some small concerns/questions:
- Why clerk is the only tool that makes somehow possible to go from clj -> cljs?
- Could this add some confusion since will affect all clojure-lsp users, like people who have both namespaces one for clj and another for cljs and find definition goes to the wrong one? 
- Adding a test may help understand that better, WDYT?

Reply:
> Why clerk is the only tool that makes somehow possible to go from clj -> cljs?

I don't think Clerk is the only tool. We use the ability to delay evaluation in various projects, e.g. the frontend sends form to the backend to perform effects similar to how Clerk sends forms representing viewer functions to the browser. The separation of `read` and `eval` is a fundamental mechanism in a lisp and I'd love to see lsp support it better.

> Could this add some confusion since will affect all clojure-lsp users, like people who have both namespaces one for clj and another for cljs and find definition goes to the wrong one?

While I can't say this will never happen I would argue that the bug to fix would then be not finding the Clojure definition. As it's now implemented its only using the ClojureScript variant as a fallback.